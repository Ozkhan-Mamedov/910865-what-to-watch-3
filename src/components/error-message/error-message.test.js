import React from "react";
import renderer from "react-test-renderer";

import ErrorMessage from "./error-message";

it(`ErrorMessage component renders correctly`, () => {
  const tree = renderer
    .create(<ErrorMessage errorMessage={`message`}/>)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
