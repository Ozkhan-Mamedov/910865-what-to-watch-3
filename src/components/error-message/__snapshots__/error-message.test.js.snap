// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorMessage component renders correctly 1`] = `
<div
  className="user-page"
>
  <header
    className="page-header user-page__head"
    style={
      Object {
        "marginBottom": "30vh",
      }
    }
  >
    <div
      className="logo"
    >
      <a
        className="logo__link"
      >
        <span
          className="logo__letter logo__letter--1"
        >
          W
        </span>
        <span
          className="logo__letter logo__letter--2"
        >
          T
        </span>
        <span
          className="logo__letter logo__letter--3"
        >
          W
        </span>
      </a>
    </div>
  </header>
  <div
    className="sign-in__message"
    style={
      Object {
        "minHeight": "33vh",
      }
    }
  >
    <p
      style={
        Object {
          "fontSize": "44px",
          "textAlign": "center",
        }
      }
    >
      message
    </p>
  </div>
  <footer
    className="page-footer"
  >
    <div
      className="logo"
    >
      <a
        className="logo__link logo__link--light"
        href={null}
      >
        <span
          className="logo__letter logo__letter--1"
        >
          W
        </span>
        <span
          className="logo__letter logo__letter--2"
        >
          T
        </span>
        <span
          className="logo__letter logo__letter--3"
        >
          W
        </span>
      </a>
    </div>
    <div
      className="copyright"
    >
      <p>
        Â© 2019 What to watch Ltd.
      </p>
    </div>
  </footer>
</div>
`;
